client/reader/full-post/index.jsx

```diff
@@ -53,6 +53,7 @@ var CommentButton = require( 'components/comment-button' ),
 
 import PostExcerpt from 'components/post-excerpt';
 import { getPostTotalCommentsCount } from 'state/comments/selectors';
+import GalleryView from 'components/tinymce/plugins/wpcom-view/gallery-view';
```

Comment:
this scares me a little. @aduth should we move the guts of gallery view elsewhere, maybe to the root of `components`?